openproductapiclient/models.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+
+from .client import Client
+
+logger = logging.getLogger(__name__)
+
+
+class Configuration(SingletonModel):
+    """
+    The Open Product API configuration to retrieve and render Open Product data.
+    """
+
+    open_product_api_service = models.ForeignKey(
+        "zgw_consumers.Service",
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
+        related_name="open_product_api_service",
+    )
+
+    open_product_types_api_service = models.ForeignKey(
+        "zgw_consumers.Service",
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
+        related_name="open_product_types_api_service",
+    )
```

Comment:
Be careful of `CASCADE`: if I delete one service but not the other, it seems more sensible to me that my `Configuration` still exists, but with one of the fields nulled:

```suggestion
    open_product_api_service = models.ForeignKey(
        "zgw_consumers.Service",
        on_delete=models.SET_NULL,
        null=True,
        blank=True,
        related_name="open_product_api_service",
    )

    open_product_types_api_service = models.ForeignKey(
        "zgw_consumers.Service",
        on_delete=models.SET_NULL,
        null=True,
        blank=True,
        related_name="open_product_types_api_service",
    )
```